operators/stackgres/1.4.3/metadata/annotations.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+<<<<<<< HEAD:operators/mongodb-atlas-kubernetes/1.7.0/metadata/annotations.yaml
```

Comment:
@teoincontatto please fix these `git conflict`  lines

Reply:
Sorry for the garbage. Should be fixed now!